runironic-inspector.sh

```diff
@@ -31,10 +26,9 @@ mkdir -p /shared/log/ironic-inspector
 
 cp $CONFIG $CONFIG.orig
 
-crudini --set $CONFIG ironic endpoint_override http://$PROVISIONING_IP:6385
-crudini --set $CONFIG service_catalog endpoint_override http://$PROVISIONING_IP:5050
+crudini --set $CONFIG ironic endpoint_override http://$IRONIC_HOST_URL:6385
+crudini --set $CONFIG service_catalog endpoint_override http://$IRONIC_HOST_URL:5050
```

Comment:
Is it `IRONIC_HOST_URL` or `IRONIC_URL_HOST`?

Reply:
Thanks, nice catch - it's fixed.